apps/gro/fields/address-outside.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  'address-text-one': {
+    validate: ['required'],
+  }
+};
```

Comment:
But they have the same key, they are identical, this is actually invalid JS as you are assigning the same key to the fields object in index.js. I'm surprised this isn't throwing up an error